### PR TITLE
UIEH-1188: Jest+RTL > Test PackageShowRoute

### PR DESCRIPTION
**Pull request by mariiaschv**
## Purpose
Add tests for `PackageShowRoute` component.

### Component code coverage
image

## Issues
[UIEH-1188](https://issues.folio.org/browse/UIEH-1188)

### CODE REVIEW

**Review by Svitlana-Korol — APPROVED** [review]

File: src/routes/package-show-route/package-show-route.test.js [hunk 1]
```diff
@@ -0,0 +1,699 @@
...
+
+    expect(getByText('PackageShowRoute component')).toBeDefined();
+  });
+
+  it('should handle getPackage', () => {
+    renderPackageShowRoute({
+      getPackage: mockGetPackage,
+    });
+
+    expect(mockGetPackage).toHaveBeenCalled();
```
Comment (BogdanDenis) [inline src/routes/package-show-route/package-show-route.test.js:287]: Please check that this function is called with correct id

File: src/routes/package-show-route/package-show-route.test.js [hunk 2]
```diff
@@ -0,0 +1,699 @@
...
+
+    expect(mockGetProxyTypes).toHaveBeenCalled();
+  });
+
+  it('should handle getProvider', () => {
+    renderPackageShowRoute({
+      getProvider: mockGetProvider,
+    });
+
+    expect(mockGetProvider).toHaveBeenCalled();
```
Comment (BogdanDenis) [inline src/routes/package-show-route/package-show-route.test.js:303]: Please check that this function is called with correct id

File: src/routes/package-show-route/package-show-route.test.js [hunk 3]
```diff
@@ -0,0 +1,699 @@
...
+        match: {
+          ...match,
+          params: {
+            ...match.params,
+            packageId: 'new-package-id',
+          },
+        },
+      }));
+
+      expect(mockGetPackage).toHaveBeenCalled();
```
Comment (BogdanDenis) [inline src/routes/package-show-route/package-show-route.test.js:443]: Please check that the function is called with new id

File: src/routes/package-show-route/package-show-route.test.js [hunk 4]
```diff
@@ -0,0 +1,699 @@
...
+        },
+      });
+
+      fireEvent.click(getAllByTestId('search-badge')[0]);
+      fireEvent.change(getByRole('searchbox', { name: 'ui-eholdings.search.enterYourSearch' }), {
+        target: { value: 'Title name' },
+      });
+      fireEvent.click(getByRole('button', { name: 'ui-eholdings.label.search' }));
+
+      expect(mockGetPackageTitles).toHaveBeenCalled();
```
Comment (BogdanDenis) [inline src/routes/package-show-route/package-show-route.test.js:496]: Please check that this is called with correct search params

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

**Review by BogdanDenis — APPROVED** [review]

**Pull request merged**